services/user/Permissions/ui/src/permissions/db.ts

```diff
@@ -0,0 +1,40 @@
+import { OAUTH_REQUEST_KEY } from "@/constants";
+import { getSupervisor } from "@psibase/common-lib";
+import { z } from "zod";
+
+const supervisor = getSupervisor();
+
+const PermsOauthRequest = z.object({
+    id: z.string(),
+    user: z.string(),
+    caller: z.string(),
+    callee: z.string(),
+});
```

Comment:
@Velua and I agreed to start using a `z` prefix for zod schemas: `zPermsOauthRequest`. That way, if it's exported/passed around, it's easy to identify what is the type and what is the zod schema.